Views/hello.html

```diff
@@ -0,0 +1,74 @@
+<!DOCTYPE html>
+<html>
+ <head>
+   <meta charset="UTF-8">
+   <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.3.1/jquery.min.js"></script>
+   <script>
+    $(document).ready(function(){
+        $('#postButton').click(function(){
+            $.post("/hello",
+            {
+              name: "JSONPost"
+            },
```

Comment:
This is inconsistent with the urlencoded POST, and the data it is POSTing is hidden from the user - could we not allow the user to enter some JSON in a box (it could have the data above as the default value)?
This would also allow someone to deliberately send malformed input and observe that it is rejected.